openSpotServer/carDetection/carDetectionModel.py

```diff
@@ -75,80 +76,57 @@ def find_cars(image_path):
     # Load pre-trained model
     model.load_weights(coco_model_path, by_name=True)
 
-    parked_car_boxes = None
     rgb = cv2.imread(image_path)
-    x = 0
-    y = 1000
-    h = 3000
-    w = 3000
-    crop = rgb[y:y + h, x:x + w]
+    crop = rgb[crop_y:crop_y+crop_h, crop_x:crop_x+crop_w]
 
     # Run the image through the Mask R-CNN model to get results.
     results = model.detect([crop], verbose=0)
 
     # Mask R-CNN assumes we are running detection on multiple images.
     # We only passed in one image to detect, so only grab the first result.
```

Comment:
We can prob start getting rid of these comments

Reply:
ill leave them in to help people understands the code